doc/sphinx/proofs/automatic-tactics/auto.rst

```diff
@@ -237,49 +232,51 @@ the optional tactic of the ``Hint Rewrite`` command.
 
 .. example:: MacCarthy function
 
-   .. coqtop:: in reset
+   This example requires the Stdlib library.
```

Comment:
That said, since the very first sentence of the example is `From Stdlib Require ...`, I guess it is redundant.

Reply:
OK, but the current wording leaves the reader wondering why that's relevant and what they should do about it.  Can you replace it with a sentence that identifies the issue and what the user may need to do to get Stdlib?  If this applies to more than one example, this info should (also) be in a more visible place, perhaps in the [Building Rocq Projects](https://coq.inria.fr/doc/V8.20.0/refman//practical-tools/utilities.html) section.

Reply:
One way to address this would be to link to a section under "The Coq libraries" which explains what is the Stdlib and how to install it (or a more generic section about the Coq Platform, since we plan to depend on more libraries from the Coq Platform).

Reply:
We may indeed improve it in the future with further links. This is definitely good enough for now.